package-lock.json

```diff
@@ -190,9 +190,15 @@
       }
     },
     "fugu-api-data": {
-      "version": "1.4.1",
-      "resolved": "https://registry.npmjs.org/fugu-api-data/-/fugu-api-data-1.4.1.tgz",
-      "integrity": "sha512-vIgJiT3fWIZOFbzG4+rbPfQRbgW/3ZqCnywc1oq5YzNisl5NY4U9ucg6b9ruCGqVjQVExlJGeDrtz/RiCYCX/A==",
+<<<<<<< HEAD
```

Comment:
Looks like a merge conflict?

Reply:
Oh, sorry, recreated the file now. PTAL.